server/plugin/api.go

```diff
@@ -293,6 +369,11 @@ func (p *Plugin) testAPI(w http.ResponseWriter, r *http.Request) {
 		w.Write([]byte(err.Error()))
 		return
 	}
+	// TODO: for testing purposes, remove later
+	mattermostUserID := r.Header.Get(constants.HeaderMattermostUserIDAPI)
+	u, _ := p.Store.LoadUser(mattermostUserID)
+	t, _ := p.ParseAuthToken(u.AccessToken)
```

Comment:
Why not hanling errors?

Reply:
removed this part, as it was for testing only